### PR TITLE
Rev groqflow to test version 3.0.2. Eliminate all keras references from docs and test.

### PR DESCRIPTION
**Pull request by jeremyfowers**
TF/Keras was continuing to cause issues for beta testers simply by being part of the python environment. `groqflow 3.0.2` makes the `tensorflow-cpu` package optional, and this update to `mlagility` takes advantage of that to exclude TensorFlow entirely.

Hopefully this will help a lot with the environment/deps issues we've been facing.

### CODE REVIEW

File: test/analysis.py [hunk 1]
```diff
@@ -268,18 +229,6 @@ def test_04_build(self):
         )
         assert np.array_equal(output, (2, 0, 1))
 
-    def test_05_build_keras(self):
```
Comment (danielholanda) [inline test/analysis.py:271]: Renumber tests

**Review by danielholanda — APPROVED** [review]

File: test/analysis.py (hunk 1 above)
Comment (jeremyfowers) [reply test/analysis.py:271]: Done!
